### DISCUSSION TITLE
[BUG] <title>I deployed according to the official immich The face data in the image cannot be recognized, and the picture type cannot be marked.

### DISCUSSION THREAD

**Discussion by JinBiLianShao** [immich-app/immich#6266] [General]
### The bug

Image tagging has never worked. I have turned on image recognition related functions in settings, but portrait recognition has always left an empty log with no obvious errors.

### The OS that Immich Server is running on

Synology nas docker

### Version of Immich Server

v1.91.4

### Version of Immich Mobile App

v1.91.4

### Platform with the issue

- [X] Server
- [X] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
Pure official document, no modifications made
```

### Your .env content

```Shell
Official document, unmodified
```

### Reproduction steps

```bash
Deploy according to official steps
```

### Additional information

The network environment is in China. Is it possible that the Great Firewall of China blocks image recognition related functions?

**Comment by schuhbacca** [comment]
I have seen others report that the downloading of the ML Models is blocked in China. Can you check the ML Docker Logs and see what it says?

**Reply by JinBiLianShao** [reply]
If I download the model manually, in which directory should I place the model? Could you please give me a link to download the model Buddha? Thank you so much

**Reply by JinBiLianShao** [reply]
If I download the model manually, in which directory should I place the model? Could you please give me a link to download the model Buddha? Thank you so much

**Comment by alextran1502** [comment]
If you are in China, you won't be able to download the models

**Reply by JinBiLianShao** [reply]
If I download the model manually, in which directory should I place the model? Could you please give me a link to download the model Buddha? Thank you so much
